operator/src/main/java/dev/responsive/k8s/crd/ResponsivePolicy.java

```diff
@@ -29,4 +29,6 @@
 @Singular("responsivepolicy")
 public class ResponsivePolicy extends CustomResource<ResponsivePolicySpec, ResponsivePolicyStatus>
     implements Namespaced {
+
+  private static final long serialVersionUID = 0;
```

Comment:
we should generate a real serial version UID (I think IntelliJ can do this for you)

Reply:
oh yeah, it can. nice! done